octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5676,6 +5686,14 @@ msgstr "Daemon offline"
 msgid "Reset daemon"
 msgstr "Dämon zurücksetzen"
 
+# ribbon cable =?
+# Flachbandkabel?
+#~ msgid ""
+#~ "Remember to turn off your Mr Beam if you disconnect the ribbon cable!"
+#~ msgstr ""
+#~ "Denke daran, Deinen Mr Beam auszuschalten, wenn Du das Flachbandkabel "
+#~ "abziehst!"
```

Comment:
why is this commented out? Why aren't we then deleting it? I see it was deleted from the message.pot

Reply:
This is done by Poedit, I think to keep old messages, but we could clean this whole section